pom.xml

```diff
@@ -87,6 +98,19 @@
 				<type>pom</type>
 				<scope>import</scope>
 			</dependency>
+
+			<!-- Solve convergence errors -->
+			<dependency>
+				<groupId>com.sun.istack</groupId>
+				<artifactId>istack-commons-runtime</artifactId>
+				<version>3.0.12</version>
+			</dependency>
+			<dependency>
```

Comment:
why is the OSGi dependency required?

Reply:
Transitive dependency from some camel-stuff. Also this is dependencyManagement section, so I am just resolving the convergence error of two different versions being dependent on.